README.md

```diff
@@ -10,7 +10,39 @@ This is a helper package that automatically installs and updates the [Angular La
 * Restart Sublime.
 
 
-
 ## Configuration
 
 Open configuration file using command palette with `Preferences: LSP-angular Settings` command or opening it from the Sublime menu (`Preferences > Package Settings > LSP > Servers > LSP-angular`).
+
+
+## Different Angular Version
```

Comment:
```suggestion
## Versioning

The language service extension relies on the `@angular/language-service` and `typescript` packages for its backend. `@angular/language-service` is always bundled with the extension, and is always the latest version at the time of the release. 
```

it would be good to note that this plugin should always ship with the latest language server.

Reply:
@princemaple do you think that adding a paragraph below versioning would be good?

> The language service extension relies on the `@angular/language-service` and `typescript` packages for its backend. `@angular/language-service` is always bundled with the extension, and is always the latest version at the time of the release.

Reply:
You meant language-server? Sure!